UnityComponent/DoomMaze/Assets/Scenes/AuthScene.unity

```diff
@@ -1153,6 +1184,87 @@ CanvasRenderer:
   m_PrefabAsset: {fileID: 0}
   m_GameObject: {fileID: 1137094532}
   m_CullTransparentMesh: 0
+--- !u!1 &1373453336
+GameObject:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  serializedVersion: 6
+  m_Component:
+  - component: {fileID: 1373453337}
+  - component: {fileID: 1373453338}
+  m_Layer: 0
+  m_Name: BgImage
```

Comment:
Also, is there any possibility that the screen displayed to the user will be larger than the image background?  I noticed the image doesn't fill the whole display port in the editor.

Reply:
The bg image was originally filling the whole camera view in the Unity Editor for me but when I re-cloned the repo, I saw the bg image offsets were incorrect.  I pushed a fix for this.

Reply:
Nice!